### DISCUSSION TITLE
Is it possible? to adjust the table display responsive column

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#2187] [Feature requests]
how to adjust the table responsive column. Same as the example below.

[https://www.npmjs.com/package/react-super-responsive-table](https://www.npmjs.com/package/react-super-responsive-table)

**Comment by rtivital** [comment]
Table component is used just to provide styles, it does not have any extra features. You can implement this feature on yourself or use  a third-party package.
